README.md

```diff
@@ -21,3 +21,4 @@ you can subscribe to.
 
 If you believe you have found a bug, please report it using the [GitHub issue tracker](https://github.com/thephpleague/omnipay-common/issues),
 or better yet, fork the library and submit a pull request.
+ 
```

Comment:
This is just a request for adding my driver to the list of officially supported gateways. Look at https://github.com/igaponov/omnipay-wirecard please.